vicodyn/include/cocaine/vicodyn/balancer/utils.hpp

```diff
@@ -0,0 +1,31 @@
+#include <random>
+#include <vector>
+
+namespace cocaine {
+namespace vicodyn {
+namespace balancer {
+
+auto uint64_rand() -> uint64_t {
+    static thread_local std::mt19937 gen;
```

Comment:
This will produce the same sequence of random numbers. Why don't you use `random_device` to seed it?

Reply:
fixed